apps/web/src/app/(private)/projects/[projectId]/versions/[commitUuid]/providers/LatteRealtimeUpdatesProvider.tsx

```diff
@@ -0,0 +1,13 @@
+'use client'
+
+import { useLatteProjectChanges, useLatteThreadUpdates } from '$/hooks/latte'
+import { memo, ReactNode } from 'react'
+
+export const LatteRealtimeUpdatesProvider = memo(
+  ({ children }: { children: ReactNode }) => {
+    useLatteThreadUpdates()
+    useLatteProjectChanges()
+
+    return children
+  },
+)
```

Comment:
technically not a provider I guess since it does not start a context but I mean you get the idea.